src/liballoc/collections/vec_deque.rs

```diff
@@ -144,11 +144,21 @@ impl<T: Clone> Clone for VecDeque<T> {
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl<#[may_dangle] T> Drop for VecDeque<T> {
     fn drop(&mut self) {
+        struct Dropper<'a, T>(&'a mut [T]);
```

Comment:
Let's leave a comment here that discusses what this `Dropper` is for.

Reply:
Done. I hope this makes it more clear.